scripts/publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+BUILD_NR=${1}
```

Comment:
generate a half-decent error message if `${1}` is undefined or empty

```suggestion
BUILD_NR=${1?:Must specify build number as first parameter}
```